doc/source/reference/series.rst

```diff
@@ -214,11 +214,17 @@ Missing data handling
 .. autosummary::
    :toctree: api/
 
-   Series.isna
-   Series.notna
+   Series.backfill
```

Comment:
@jorisvandenbossche is there any way we can indicate aliases here? (should we just not list them)?